cli/pcluster/config/pcluster_config.py

```diff
@@ -344,18 +345,38 @@ def __check_account_capacity(self):  # noqa: C901
 
         try:
             # Check for insufficient Account capacity
-            subnet_id = vpc_section.get_param_value("compute_subnet_id")
-            if not subnet_id:
-                subnet_id = vpc_section.get_param_value("master_subnet_id")
+            compute_subnet = vpc_section.get_param_value("compute_subnet_id")
+            master_subnet = vpc_section.get_param_value("master_subnet_id")
+            if not compute_subnet:
+                compute_subnet = master_subnet
 
             placement_group = cluster_section.get_param_value("placement_group")
+            disable_hyperthreading = cluster_section.get_param_value("disable_hyperthreading")
+            master_vcpus = get_instance_vcpus(self.region, master_instance_type)
+            compute_vcpus = get_instance_vcpus(self.region, compute_instance_type)
 
             boto3.client("ec2").run_instances(
-                InstanceType=instance_type,
+                InstanceType=master_instance_type,
```

Comment:
Could you add two comments before the run_instances describing why we are doing two tests (I know it is to test master and compute but we have to check the code to find the differences).

Then we can define the variables for `CpuOptions` and `Placement` in a single place before the two run_instances.